exam/src/main/java/in/testpress/exam/ui/TestActivity.java

```diff
@@ -579,12 +580,40 @@ private void displayTestFragment(Bundle arguments) {
         if (getSupportActionBar() != null) {
             getSupportActionBar().hide();
         }
+
+        if (examInstructionExists()){
+            displayExamInstruction(arguments);
```

Comment:
Why all arguments are passed ? Pass only necessary arguments

Reply:
We're passing those arguments to the Exam fragment after the instruction page.

Reply:
ok